cardano-config/src/Cardano/Config/Shelley/Orphans.hs

```diff
@@ -81,7 +85,7 @@ instance Crypto crypto => FromJSON (ShelleyGenesis crypto) where
         <*> obj .: "MaxHeaderSize"
         <*> obj .: "GenDelegs"
         <*> obj .: "InitialFunds"
-
+        <*> pure emptyGenesisStaking  --TODO
```

Comment:
This turns out to  be quite big, and ends up needing extra conversion functions in the ledger for opaque types. So these changes cannot be included in this PR.